pkgs/top-level/php-packages.nix

```diff
@@ -14,8 +14,9 @@ let
     });
 
   isPhp73 = pkgs.lib.versionAtLeast php.version "7.3";
+  isPhp74 = pkgs.lib.versionAtLeast php.version "7.4";
 
-  apcu = buildPecl {
+  apcu = assert !isPhp74; buildPecl {
```

Comment:
apcu in this version is already compatible https://pecl.php.net/package-info.php?package=APCu&version=5.1.17

Reply:
Updated package to 5.1.18 version.